### ISSUE TITLE
[Static Site] Elements with duplicated IDs 

### ISSUE THREAD

**Issue by johanna-skylight** [CDCgov/prime-simplereport#5808]
## Area or Component

Page [SimpleReport data catalog](https://www.simplereport.gov/getting-started/public-health-departments/simplereport-data-catalog/). Accordion Tab Examples.

## Quick Summary

Below headings in the example tab are using the same ID of other elements in the page.

Patient Information (H3)
Ask on order entry (H4)
Test information (H3)
Results (H4)

### How to Replicate

Go to the page https://www.simplereport.gov/getting-started/public-health-departments/simplereport-data-catalog/ and run the axe scan.

## Accessibility Violation

id attribute value must be unique https://dequeuniversity.com/rules/axe/4.7/duplicate-id?application=AxeChrome

## Action Requested

Rename the IDs of the headings listed above and make sure that if those elements are referenced in other areas the functionality does not brea if the IDs are using to implement some JS behavior or to reference other aria attributes.

## Screenshots (optional)

![Screenshot 2023-05-15 at 2 04 57 PM](https://github.com/CDCgov/prime-simplereport/assets/103958711/a7e6feb7-cae0-4ffa-80f5-8d0abedd76ef)
